interface/forms/reviewofs/new.php

```diff
@@ -28,787 +28,619 @@
 
     <?php Header::setupHeader();?>
```

Comment:
(dang, that was a tough merge :) ; i just went through the merge also).
Just 2 more changes needed in this script:
```
<div class="btn-group oe-opt-btn-group-pinch" role="group">
```
to
```
<div class="btn-group" role="group">
```
and then
```
<button type="button" class="btn btn-secondary btn-cancel oe-opt-btn-separate-left" onclick="top.restoreSession(); parent.closeTab(window.name, false);"><?php echo xlt('Cancel');?></button>
```
to 
```
<button type="button" class="btn btn-secondary btn-cancel" onclick="top.restoreSession(); parent.closeTab(window.name, false);"><?php echo xlt('Cancel');?></button>
```

Then this PR will be ready for codebase.

Reply:
Update the commit.